src/task.rs

```diff
@@ -2052,6 +2125,11 @@ impl TaskMetrics {
         self.total_slow_poll_count as f64 / self.total_poll_count as f64
     }
 
+    /// The ratio of tasks exceeding [`long_delay_threshold`][TaskMonitor::long_delay_threshold].
```

Comment:
Can you follow the documentation pattern of [`slow_poll_ratio`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMetrics.html#method.slow_poll_ratio) here?